modules/imaging_uploader/php/imaging_uploader.class.inc

```diff
@@ -294,7 +309,7 @@ class Imaging_Uploader extends \NDB_Menu_Filter_Form
             ///////////////////////////////////////////////////////////////////////
             $pcv  = $pscid . "_" . $candid . "_" . $visit_label;
             $pcvu = $pcv . "_";
-            if ((!preg_match("/{$pcv}\.(zip|tgz|tar.gz)/", $file_name))
+            if ((!preg_match("/^{$pcv}\.(zip|tgz|tar.gz)/", $file_name))
```

Comment:
so all along we never restricted the *start* of the filename to be PSCID_....?
then this is a bug fix as well.

Reply:
can this specific line change be put in: https://github.com/aces/Loris/pull/3547?